newrelic/core/environment.py

```diff
@@ -170,6 +170,20 @@ def environment_settings():
         if hasattr(uvicorn, "__version__"):
             dispatcher.append(("Dispatcher Version", uvicorn.__version__))
 
+    if not dispatcher and "hypercorn" in sys.modules:
+        dispatcher.append(("Dispatcher", "hypercorn"))
+        hypercorn = sys.modules["hypercorn"]
+
+        if hasattr(hypercorn, "__version__"):
+            dispatcher.append(("Dispatcher Version", hypercorn.__version__))
+        else:
```

Comment:
Could you add a comment above this section explaining what's going on here? I know primarily because you just talked about this in the presentation today but I don't think it's going to be as obvious years later when someone looks at this code again. It would also be helpful to provide the version(s) for which this special code is needed so at potentially a much later date we can remove this special code forever when we no longer need to support the version that requires it.

Reply:
Done